sql/sql_admin.cc

```diff
@@ -711,7 +711,7 @@ static bool mysql_admin_table(THD* thd, TABLE_LIST* tables,
             protocol->store(&table_name, system_charset_info);
             protocol->store(operator_name, system_charset_info);
             protocol->store(&error_clex_str, system_charset_info);
-            length= my_snprintf(buff, sizeof(buff),
+            length= my_snprintf(buff, sizeof(buff), "%s",
                                 ER_THD(thd, ER_PARTITION_DOES_NOT_EXIST));
```

Comment:
Per Zulip chat, I’ve switched copy-with-truncation of constants to use `strmake`.
```suggestion
            length= strmake(buff, ER_THD(thd, ER_PARTITION_DOES_NOT_EXIST),
                            sizeof(buff)-1) - buff;
```
I’ll batch-commit them later as I review all `my_snprintf` usages.

Reply:
I split these extra work to #3429.
It will likely conflict with the old patch here; in which case, that commit’s `strmake`s take precedence.